static/app/views/eventsV2/queryList.tsx

```diff
@@ -244,12 +244,20 @@ class QueryList extends React.Component<Props> {
             });
           }}
           renderGraph={() => (
-            <MiniGraph
-              location={location}
-              eventView={eventView}
+            <Feature
               organization={organization}
-              referrer={referrer}
-            />
+              features={['connect-discover-and-dashboards', 'dashboards-edit']}
```

Comment:
Out of curiosity why does this feature check need dashboards-edit too?

Reply:
You're right `dashboards-edit` isn't needed here 😄 . Updated!